gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "0.44.9",
+  "version": "0.45.1",
```

Comment:
Why .1?

Reply:
> Why .1?

Brain fart, and expecting RSC-93 to be merged first